SVRN/donations.yml

```diff
@@ -94,4 +94,19 @@
     - SVRN/Donation_Receipt_8476.jpg 
   charity_tx_id: fc27ae6bb5928e35795de07d3ccc53a361f923930526a6c6d79944ddb56438da
   
- 
\ No newline at end of file
+- ticker: SVRN
+  announcement_url: https://sleepingnatives.org/donations.html
+  recipient: 
+    name: Sleeping Natives Productions
+    address: 1611, Crenshaw Blvd., Suite 165, Torrance, CA, USA
+    tel: +1 310 961 4659
+    contact: philip@sleepingnatives.org
+  ada: 3700.00000
+  usd: 4,255.00
+  timestamp: 1648129122
+  files:
+    - SVRN/7th_SVRN_Rewards_Funds_Withdrawal.jpg
+    - SVRN/7th_SVRN_Pledge_Increase.jpg
+    - SVRN/Donation_Receipt_8477.jpg 
+  charity_tx_id: ff3b250b822dd591f298bff5c93028e90eeb19210d54c44e6b8fb5932a9ac568
+  
```

Comment:
I think you need to remove this blank line from the bottom of the donations.yml file. For some reason the automated checks are failing to validate the file but I am not seeing any YAML file validation issues.

Reply:
Hi, sorry for the late reply - I've updated the branch (before making the suggested edit) - and it automatically appears to be accepted. Could you please see again if it can merge?

If not, I'll remove the blank line at the bottom (but yeah, it should be valid I think...) Thanks, Philip